docs/how-to-guides/convert-changelog-to-table.md

```diff
@@ -15,19 +15,17 @@ You have a stream of events that represent a series of changes, known as a chang
 ## In action
 
 ```sql
-CREATE TABLE t1 AS
```

Comment:
I think we need a larger change here (depending on what we want to convey \cc @MichaelDrogalis).

> You have a stream of events that represent a series of changes, known as a changelog. 

If this "stream of event" is just stored in a topic (and there not necessarily a STREAM defined), we can change the query to 
```
CREATE SOURCE TABLE t1 <schemaDefinition> WITH (kafka_topics='changelog-topic',...);
```


Instead of specifying a query with an input STREAM.